### PR TITLE
Updated tldr-pages.yml

### PR DESCRIPTION
**Pull request by filipe-gomes**
Related to #1444

Added mention to hacktoberfest, removed "help wanted."

### CODE REVIEW

File: _data/projects/tldr-pages.yml [hunk 1]
```diff
@@ -15,8 +15,8 @@ tags:
 - shell
 - cli
 upforgrabs:
-  name: help wanted
-  link: https://github.com/tldr-pages/tldr/labels/help%20wanted
+  name: Hacktoberfest
```
Comment (shiftkey) [inline _data/projects/tldr-pages.yml]: This value is case-sensitive when looking for it using the GitHub API, so we need it to match:

```suggestion
  name: hacktoberfest
```

**Review by shiftkey — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
It looks like the renamed project (no diff displaying) has `Help wanted` as the label, but it should be `help wanted`:

https://github.com/up-for-grabs/up-for-grabs.net/blob/a20c8e81698aaec96e00161813204d74459fe139/_data/projects/COMP30022-frontend.yml#L11-L12

**Comment by filipe-gomes** [comment]
Thanks, I will fix that in the PR.

**Review by shiftkey — APPROVED** [review]
Thanks @filipe-gomes!

**Comment by shiftkey** [comment]
@all-contributors please add @filipe-gomes for content

**Pull request merged**